packages/components/src/TCRUserDashboard/DashboardStyledComponents.tsx

```diff
@@ -78,7 +78,95 @@ export const StyledDashboardActivityDescription = styled.div<StyledDashboardActi
 `;
 
 export const StyledUserInfo = styled.div`
-  width: 277px;
+  width: 333px;
+  ${mediaQueries.MOBILE} {
```

Comment:
FYI `mediaQueries.MOBILE` is `max-width: 1030px`, `mediaQueries.MOBILE_SMALL` is `max-width: 640px` not sure you ant the larger size here. We should name these better when we make updates to style guide.

Reply:
`MOBILE` is the point where we change the layout of the page (stack components on top  of each other), so I think it's correct here